### DISCUSSION TITLE
Running a command when a download finishes

### DISCUSSION THREAD

**Discussion by aduros** [qutebrowser/qutebrowser#6898] [Q&A] [answered]
Hi all, is there a way to run a custom script whenever a download finishes? I'd like to have something that sends a desktop notification when a download finishes, and maybe do some other processing.

Thanks :smile:

**Comment by The-Compiler** [comment, marked as answer]
Not from qutebrowser's side, see #30 and #35. However, you might be able to use something like `inotifywatch` to run a command every time a new file appears in your download directory outside of qutebrowser.
